packages/vscode-extension/src/handlers.ts

```diff
@@ -1645,7 +1647,7 @@ export async function showError(e: UserError | SystemError) {
       },
     };
 
-    const button = await window.showErrorMessage(`[${errorCode}]: ${e.message}`, help);
```

Comment:
can we keep the current behavior by using 'message' as notification if notificationMessage is undefined?